dist/main.js

```diff
@@ -13022,7 +13062,7 @@ angular.module('znk.infra-web-app.onBoarding').run(['$templateCache', function (
                 promoCodeStatusText[INVALID] = INVALID;
 
                 promoCodeSrv.checkPromoCode = function (promoCode, appContext) {
-                    var firebaseAppScopeName =  backendData[appContext].firebaseAppScopeName;
+                    var firebaseAppScopeName = backendData[appContext].firebaseAppScopeName;
```

Comment:
in one place you use firebaseAppScopeName and other appName - > has to be consistent